.github/workflows/build_go_library.yaml

```diff
@@ -67,8 +67,26 @@ jobs:
           aws configure set aws_secret_access_key ${{ secrets.PROD_ASSET_SECRET_KEY }}
           aws configure set region ap-south-1
 
+      - name: Setup Go environment
+        uses: actions/setup-go@v4
+        with: 
+            go-version: 1.21.5
+    
+      - run: |
+          go version
+          echo $GOPATH
+          echo $GOROOT
+          echo $PATH
+          cd /root/ 
+          export GOROOT=/home/github/actions-runner/_work/_tool/go/1.21.5/x64
```

Comment:
This is fragile code. Anyone making version update to golang will miss to update this path. Also its used at multiple places. 

- Make golang version as a variable  
- Find what is the Github Action variable for path until `_work` or until `_tool` 
- Use both variables to construct final GOROOT path.